aries-site/src/examples/components/table/TableSortable.js

```diff
@@ -0,0 +1,173 @@
+  Box,
+  DataTable,
+  Heading,
+  Meter,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+
+const data = [
+  {
+    id: 1,
+    string: 'durian',
+    numeric: 42,
+    date: '01/25/1915',
+    percent: 5,
+    status: 'ok',
+  },
+  {
+    id: 2,
+    string: 'cherimoya',
+    numeric: 3.14159,
+    date: '12/07/1941',
+    percent: 99,
+    status: 'ok',
+  },
+  {
+    id: 3,
+    string: 'mangosteen',
+    numeric: 3,
+    date: '03/21/1965',
+    percent: 33,
+    status: 'warning',
+  },
+  {
+    id: 4,
+    string: 'jackfruit',
+    numeric: 2.7182,
+    date: '12/15/1791',
+    percent: 25,
+    status: 'ok',
+  },
+  {
+    id: 5,
+    string: 'kiwi',
+    numeric: 1.618,
+    date: '11/03/2020',
+    percent: 67,
+    status: 'critical',
+  },
+  {
+    id: 6,
+    string: 'lemon',
+    numeric: 182282,
+    date: '07/20/1969',
+    percent: 75,
+    status: 'warning',
+  },
+  {
+    id: 7,
+    string: 'rambutan',
+    numeric: 73,
+    date: '08/06/1991',
+    percent: 50,
+    status: 'ok',
+  },
+];
+
+const columns = [
+  { property: 'string', header: 'Fruits' },
+  { property: 'numeric', header: 'Favorite Numbers', align: 'end' },
+  {
+    property: 'date',
+    header: 'Significant Dates',
+    render: datum =>
+      datum.date && new Date(datum.date).toLocaleDateString('en-US'),
+    align: 'end',
+  },
+  {
+    property: 'percent',
+    header: '% Complete',
+    render: datum => (
+      <Box pad={{ vertical: 'xsmall' }}>
+        <Meter
+          values={[{ value: datum.percent, color: 'graph-2' }]}
+          thickness="small"
+          size="small"
+        />
+      </Box>
+    ),
+  },
+  {
+    property: 'status.value',
+    header: 'Status',
+    render: datum => (
+      <Box direction="row" align="center" gap="xsmall">
+        <Box
+          height="12px"
+          width="12px"
+          background={datum.status.color}
+          round="100%"
+        />
+        <Text>{datum.status.label}</Text>
+      </Box>
+    ),
+  },
+];
+
+const formatData = dataSet =>
+  dataSet.map(datum => {
+    const adjustedDatum = { ...datum };
+    adjustedDatum.date = new Date(datum.date);
+    switch (datum.status) {
+      case 'ok':
+        adjustedDatum.status = {
+          label: datum.status,
+          value: 0,
+          color: 'status-ok',
+        };
+        break;
+      case 'warning':
+        adjustedDatum.status = {
+          label: datum.status,
+          value: 1,
+          color: 'status-warning',
+        };
+        break;
+      case 'critical':
+        adjustedDatum.status = {
+          label: datum.status,
+          value: 2,
+          color: 'status-critical',
+        };
+        break;
+      default:
+        adjustedDatum.status = {
+          label: datum.status,
+        };
+    }
+    return adjustedDatum;
+  });
+
+export const TableSortable = () => {
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <>
+      <Heading level={3}>Sortable Items</Heading>
```

Comment:
On Safari and Firefox, this example is not taking full width like it does on Chrome:
<img width="1368" alt="Screen Shot 2020-09-14 at 3 49 53 PM" src="https://user-images.githubusercontent.com/12522275/93145738-f5123480-f6a1-11ea-91ce-a1436c5d3c96.png">

Reply:
I also think in the future, we may want to switch out the data here to feel more "real world" to our use cases, but I think having these different data types is beneficial to test the sorting changes we are working on in grommet. So I think for now, they are fine.

Reply:
I agree that we should strive for that as well. I struggled to invent a real world set which lent itself nicely to the purpose of this example which is to showcase sorting by different data types. I am certainly not opposed to swapping out, just hit the limit on inventing data sets at the time.

Reply:
Addressed full-width vs. not full-width. 

For reference, HTML `<table />` is treated differently across browsers. For consistency across browsers, use either `fill="horizontal"` on `<DataTable />` or Box align.